src/services/chore.service.js

```diff
@@ -0,0 +1,58 @@
+const Chore = require("../models/chore.model");
+const Preference = require("../models/preference.model");
+const Assignment = require("../models/assignment.model");
+
+const RoomService = require("./room.service");
+
+exports.createChoreAndPreferences = async function(body) {
+  let objects = {};
```

Comment:
you can use ```const objects``` because the objects variable itself is constant. (the reference itself won't change, but some of it's fields might change which is okay)